en-US/win10/samples/ZigBeeAdapterTutorial.md

```diff
@@ -0,0 +1,203 @@
+
+You can verify that devices have by using “network discovery” feature of XCTU tool.
+
+![ZigBeeJoinNetVerif]({{site.baseurl}}/images/ZigBee/ZigBeeJoinNetVerif.png)
+
+## Set up your Raspberry Pi2
+1. perform initial set up as instructed [here](http://ms-iot.github.io/content/en-US/win10/SetupRPI.htm)
+2. plug the XBee USB dongle into the Raspberry Pi2
+3. install the FTDI driver if not already part of the Windows 10 image for Raspberry Pi2
+ 4. start a remote powershell session as administrator to the Raspberry Pi2
+ 5. copy the .inf and .sys file of the FTDI driver to c:\windows\system32 folder of the RaspBerry Pi2
+ 6. install the FTDI drivers using the following command:  `devcon.exe dp_add FTDIBUS.inf` and `devcon.exe dp_add FTDIPORT.inf`
+ 7. verify that driver works using the following command:  `devcon status "USB\VID_0403&PID_6001"`.
+You should see something like that: "_USB\VID_0403&PID_6001\... Name: FT232R USB UART Driver is running_".
+Note that PID could also be 6015 instead of 6001. This depends on the XBee Explorer USB dongle your using. 
+ 8. reboot the RaspBerry Pi2
+
+## Deploy the ZigBee adapter on your Windows 10 machine
+1. Download the ZigBeeAdapter.zip file [here](https://github.com/ms-iot/samples/blob/develop/AllJoyn/AllJoynZigBeeAdapter/ZigBeeAdapter.zip?raw=true)
+2. Navigate to the folder where you downloaded the zip file. Right click the file and “Extract All…” to the folder of your choice.
+3. Navigate to the extracted folder and open the ZigBeeAdapter.sln solution file in Visual Studio.
+4. Select the relevant target (x86, x64 or ARM) and build the solution in Visual Studio.
+Your now ready to launch it, so launch or debug HeadedAdapterApp project on desktop or if the targeted Windows 10 device has a display or launch or debug HeadlessAdapterApp if it doesn’t. 
+If needed, see instruction [here](http://ms-iot.github.io/content/en-US/win10/AppDeployment.htm) for remote debugging.
+ 
+ 
+### Known limitations of the current version of the ZigBee adapter
+- ZigBee adapter only discovers devices that are directly connected to the XBee module. 
+- ZigBee adapter only support some devices defined in ZigBee Light Link or Home Automation profiles. This means that it only implements the necessary ZCL clusters and ZDO commands to handle them. That said support for new devices and new ZCL clusters can easily be added to ZigBee adapter code.
+- ZigBee adapter doesn’t expose any method to commission ZigBee devices.
+
+## ZigBee adapter in detail 
+ZigBee adapter is written in C# and exposes ZigBee devices on AllJoyn through BridgeRT interface. ZigBee device is exposed in AllJoyn as follow:
+- Each ZigBee end point of a ZigBee device is exposed as an AllJoyn Service (bus attachment)
```

Comment:
I see them as a list in https://github.com/nicogbg/content/blob/develop/en-US/win10/samples/ZigBeeAdapterTutorial.md => don't know what to fix